### PR TITLE
added Google Ads sample integration

### PR DESCRIPTION
**Pull request by bartoszstelmaszuk**

### CODE REVIEW

**Comment by stodirascu** [comment]
My personal opinion - we should explain but not showcase how to integrate the xcframework manually. Having the binary in git is a red flag. Not to mention that it ought to always be up-to-date with development.

**Comment by stodirascu** [comment]
I thought we're adding only one xcframework, but we're adding 7. I definitely am against this. At most, I'd say we can have links to the respective repositories and tell the integrators to download them from there and leave the steps as is.
cc @DaveMiscampbell 

P.S. In general, I'd suggest we use the same approach for our xcframework, and remove it from this repo.

**Comment by DaveMiscampbell** [comment]
OK I agree with @stodirascu - let's gitignore the `Libraries` folder. 

I also think we should:
- change the top level README to just be links to the subfolders
- move the specific bits of the top level README to the subfolders
(so more like this repo - https://github.com/aws-samples/aws-cdk-examples)

And then we can add instructions for XCFrameworks in the relevant folder

File: Shared/Ads/StorytellerAdsDelegate.swift [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+        let ad = getMatchingAd(adID: data.adId)
+        switch type {
+        case .OpenedAd:
+            trackingManager.trackImpression(ad: ad)
+            trackingManager.trackEnteredViewability(adView: data.adView, ad: ad)
+        
+        case .AdActionButtonTapped:
+            trackingManager.trackClick(ad: ad)
+            
+        case .DismissedStory, .DismissedAd, .DismissedClip:
```
Comment (DaveMiscampbell) [inline Shared/Ads/StorytellerAdsDelegate.swift:30]: Let's take DismissedClip out of here

File: .gitignore [hunk 2]
```diff
@@ -72,4 +72,7 @@ fastlane/test_output
 .idea/workspace.xml
 
 # Misc
-.DS_Store
\ No newline at end of file
+.DS_Store
+
+# XCFrameworks
+XCFrameworks-Sample/StorytellerSampleApp/Libraries/
```
Comment (stodirascu) [inline .gitignore:78]: How about adding something more general, like `*.xcframework`?

File: SwiftPackageManager-Sample/StorytellerSampleApp.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -76,6 +82,11 @@
...
+		AF82186929953F5500DE31BE /* NativeAdsRequest.swift */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.swift; name = NativeAdsRequest.swift; path = ../Shared/Ads/NativeAdsRequest.swift; sourceTree = "<group>"; };
+		AF82186A29953F5500DE31BE /* Constants.swift */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.swift; name = Constants.swift; path = ../Shared/Ads/Constants.swift; sourceTree = "<group>"; };
+		AF82186B29953F5500DE31BE /* AdManager.swift */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.swift; name = AdManager.swift; path = ../Shared/Ads/AdManager.swift; sourceTree = "<group>"; };
+		AF82186C29953F5500DE31BE /* StorytellerAdsDelegate.swift */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.swift; name = StorytellerAdsDelegate.swift; path = ../Shared/Ads/StorytellerAdsDelegate.swift; sourceTree = "<group>"; };
+		AF82186D29953F5500DE31BE /* TrackingManager.swift */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.swift; name = TrackingManager.swift; path = ../Shared/Ads/TrackingManager.swift; sourceTree = "<group>"; };
```
Comment (stodirascu) [inline SwiftPackageManager-Sample/StorytellerSampleApp.xcodeproj/project.pbxproj:89]: All paths have ../
Maybe the parent folder's location needs a look at

**Pull request merged**